examples/common/utilities.adb

```diff
@@ -27,6 +34,46 @@ package body Utilities is
 
     --  ------------------------------------------------------------------------
 
+    procedure Print_GL_Array3 (Name : String; anArray : GL.Types.Single_Array) is
```

Comment:
I still don't get why you are not simply using `GL.Types.Singles.Vector3_Array`.